src/hle.c

```diff
@@ -347,6 +342,36 @@ static void non_task_dispatching(struct hle_t* hle)
 #endif
 }
 
+/* Resident evil 2 */
+static void re2_task_dispatching(struct hle_t* hle)
+{
+    const unsigned int sum =
+        sum_bytes((void*)dram_u32(hle, *dmem_u32(hle, TASK_UCODE)), 256);
+    
+    switch (sum) {
+    
+    case 0x450f:
+        resize_bilinear_task(hle);
+        return;
+    
+    case 0x3b44:
+        decode_video_frame_task(hle);
+        return;
+
+    case 0x3d84:
+        /* TODO: Nothing to emulate? */
```

Comment:
> Like I was expecting the framebuffer address (REG.VI_ORIGIN) is constantly switching between 2 addresses 0x0063e418 and 0x0076a418.
> Right now only the buffer at the first address is populated, this is done by the decode_video_frame task. The buffer at the second address is populated by another ucode which is not reversed engineered yet.
> 
> So work is not finished here.

Reply:
What is the impact of this? When I tested it seemed like everything was working properly in HLE

Reply:
2 ucodes seems to be necessary to video decoding. One is implemented with this PR, the other isn't yet. So to avoid the constant spamming of warning "unknown task ..." a rsp_break is done, which effectively bypass the non implemented ucode.